package.json

```
@@ -18,7 +19,15 @@
     "vuex": "^3.6.2"
   },
   "devDependencies": {
+    "@testing-library/vue": "^5.6.2",
     "@vue/cli-service": "~4.5.0",
+    "@testing-library/jest-dom": "^5.12.0",
+    "@vue/cli-plugin-unit-jest": "^4.5.13",
+    "@vue/babel-preset-app": "^4.5.13",
+    "@vue/test-utils": "^1.2.0",
+    "@types/jest": "^26.0.23",
+    "vue-jest": "^3.0.7",
```

```suggestion
    "@types/jest": "^26.0.23",
```
You shouldn't need those dependencies if you are using `@vue/cli-plugin-unit-jest` 🤔